indico/modules/events/sessions/templates/management/session_protection.html

```diff
@@ -0,0 +1,13 @@
+{% from 'forms/_form.html' import form_header, form_footer, form_rows, form_row, form_field %}
+
+{% block content %}
+    {{ form_header(form, id='protection-form') }}
+    {{ form_row(form.protection_mode) }}
```

Comment:
I used it because of the `ProtectionMessageField`, but since I am gonna transform it into a widget I will not need this template anymore.